sphinx_automodapi/autodoc_enhancements.py

```diff
@@ -73,66 +71,21 @@ def type_object_attrgetter(obj, attr, *defargs):
     return getattr(obj, attr, *defargs)
 
 
-if SPHINX_LT_15:
-    # Provided to work around a bug in Sphinx
-    # See https://github.com/sphinx-doc/sphinx/pull/1843
-    class AttributeDocumenter(AttributeDocumenter):
-        @classmethod
-        def can_document_member(cls, member, membername, isattr, parent):
-            non_attr_types = cls.method_types + class_types + \
-                (MethodDescriptorType,)
-            isdatadesc = isdescriptor(member) and not \
-                isinstance(member, non_attr_types) and not \
-                type(member).__name__ == "instancemethod"
-            # That last condition addresses an obscure case of C-defined
-            # methods using a deprecated type in Python 3, that is not
-            # otherwise exported anywhere by Python
-            return isdatadesc or (not isinstance(parent, ModuleDocumenter) and
-                                  not inspect.isroutine(member) and
-                                  not isinstance(member, class_types))
-
-
 def setup(app):
     # Must have the autodoc extension set up first so we can override it
     app.setup_extension('sphinx.ext.autodoc')
     # Need to import this too since it re-registers all the documenter types
     # =_=
-    import sphinx.ext.autosummary.generate
```

Comment:
we don't need this any more?

Reply:
I checked this, and in 1.3 there are a bunch of global-level `add_documenter` calls in `generate.py`, and they are all gone in 1.7 (replaced by a `main()` level invocation that isn't used if you use it as a plugin, apparently).  So unless I'm missing something subtle this was fine.  (I did delete the comment above, though, which now makes no sense).